packages/sdk/contractkit/src/wrappers/LockedGold.ts

```diff
@@ -316,11 +316,25 @@ export class LockedGoldWrapper extends BaseWrapperForGoverning<LockedGold> {
    * @param epochNumber The epoch to retrieve AccountSlashed at.
    */
   async getAccountsSlashed(epochNumber: number): Promise<AccountSlashed[]> {
-    const blockchainParamsWrapper = await this.contracts.getBlockchainParameters()
-    const events = await this.getPastEvents('AccountSlashed', {
-      fromBlock: await blockchainParamsWrapper.getFirstBlockNumberForEpoch(epochNumber),
-      toBlock: await blockchainParamsWrapper.getLastBlockNumberForEpoch(epochNumber),
-    })
+    let fromBlock: BlockNumber
+    let toBlock: BlockNumber
+
+    // TODO(L2): this is deprecated and not supported in L2
```

Comment:
im confused by these todos. are they a reminder for us to come back?

Reply:
correct we added them with leszek